packages/server/src/threads/utils.js

```diff
@@ -72,7 +72,9 @@ exports.hasExtraData = response => {
   return (
     typeof response === "object" &&
     !Array.isArray(response) &&
+    response &&
     response.data != null &&
+    response &&
```

Comment:
do we need this double `response` check?

Reply:
Yep, it's really just instead of a `response?.data` - it caught me out at runtime with the redis connector so just added some safety to it.

Reply:
I mean more around we're already doing `response &&` prior to the data check, and then again after. Is the second check here not redundant?

Reply:
Ah - yep it is, will remove the second one.